bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp

```diff
@@ -1218,6 +1219,7 @@ void BcStTest::getReservedPagesStage() {
 /////////////////////////////////////////////////////////
 
 class BcStTestParamFixture1 : public BcStTest, public testing::WithParamInterface<tuple<uint32_t, uint32_t>> {};
+class BcStTestParamFixture2 : public BcStTest, public testing::WithParamInterface<tuple<bool, uint8_t>> {};
```

Comment:
You are defining the new fixture not in the right place.
I suggest to define it just above:
`TEST_P(BcStTestParamFixture2, dstSourceSelectorPrimaryAwareness) {`

Reply:
Thank you, I've moved it to the right place.